bin/yaml/cobol.yaml

```diff
@@ -0,0 +1,17 @@
+compilers:
+  cobol:
+    cobol:
+      type: tarballs
+      compression: tar
+      dir: gnucobol
+      url: https://sourceforge.net/projects/gnucobol/files/gnucobol/3.2/gnucobol-3.2-rc2.tar.bz2
+      check_exe: bin/cobc --version
+      after_stage_script:
+        - cd {dir}
+        - ./autogen.sh
+        - ./configure
```

Comment:
notes (as pointed out in the other issue):
* autogen.sh is only reasonable when building fro a vsc checkout, otherwise it only adds build dependencies (and in the vcs checkout case you'd additionally need more tools - see the HACKING file in the source distribution/vcs)
* if the website can't handle ncurses (which I _guess_) then best disable it `-without-curses` and similar enable what is explicit expected `--with-math=gmp --with-db --with-xml2 --with-json=json-c` [not necessarily needed, configure will just use what it finds and take some extra cycles to do so)